### DISCUSSION TITLE
Login page does not redirect

### DISCUSSION THREAD

**Discussion by dynamicat** [firefly-iii/firefly-iii#6882] [General]
I have firefly latest running in docker.  When I enter the username and password and click login it does not redirect me to the main dashboard, the page stays static.  I checked in the console view and I get this:

![image](https://user-images.githubusercontent.com/12977282/212877600-7a171eeb-5168-4c17-b753-9ad8680a0e98.png)

Which seems correct.  

is there anything I should check or you know what is missing?

**Comment by JC5** [comment]
In the source of the html page, is the `<base href="">` correct?

**Reply by dynamicat** [reply]
base href on login page is:  `https://firefly.mydomain.com/">`

**Reply by dynamicat** [reply]
I am still unable to debug it.  By the way I am using Traefik proxy if that helps?

**Reply by dynamicat** [reply]
I have it working I added the following to my labels:


       ` - "traefik.frontend.headers.SSLRedirect=true"`
       ` - "traefik.frontend.headers.customRequestHeaders=X-Forwarded-Proto:https"`
       ` - "traefik.frontend.headers.SSLProxyHeaders=X-Forwarded-Proto:https"`

**Reply by JC5** [reply]
Yay glad to hear it!
